gamemodes/terrortown/gamemode/server/sv_player.lua

```diff
@@ -124,6 +124,9 @@ function GM:PlayerSpawn(ply)
 	ply:SetupHands()
 
 	SCORE:HandleSpawn(ply)
+
+	-- reset armor of player on spawn
+	ply:SetArmor(0)
```

Comment:
Very ugly to paste it anywhere. Should be used in functions that are created for reset. E.g. plymeta:ResetRoundFlags()

Reply:
I removed it at the other place, it is only in this one position. I could use a function for this if you really prefer it.

Reply:
yea, you should use the requested or any similar function :)

Reply:
okay, no problem